src/main/java/uk/gov/hmcts/reform/probate/model/forms/Address.java

```diff
@@ -15,6 +15,9 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Address {
 
+    public Address(String address) {
```

Comment:
why is this constructor needed?

Reply:
The constructor is required to allow old style address in form data to be migrated